modin/pandas/base.py

```diff
@@ -1473,17 +1473,24 @@ def kurt(self, axis=None, skipna=None, level=None, numeric_only=None, **kwargs):
         Returns:
             kurtosis : Series or DataFrame (if level specified)
         """
+        axis = self._get_axis_number(axis)
         if level is not None:
-            return self._default_to_pandas(
-                "kurt",
-                axis=axis,
-                skipna=skipna,
-                level=level,
-                numeric_only=numeric_only,
-                **kwargs,
-            )
+            from .series import Series
+
+            def applyf(x):
```

Comment:
I think this will be problematic for some query compiler implementations. We cannot assume that query compilers can consume a python function.

Reply:
Yes, but notice that `applyf` passed to the [`modin.pandas.DataFrame.aggregation`](https://github.com/modin-project/modin/blob/master/modin/pandas/base.py#L359) and [`modin.pandas.groupby.DataFrameGroupBy.aggregate`](https://github.com/modin-project/modin/blob/master/modin/pandas/groupby.py#L341) whose interfaces is guarantee to be able to obtain python function. Under the hood them both calls [`query_compiler.apply`](https://github.com/modin-project/modin/blob/master/modin/backends/base/query_compiler.py#L804) or [`query_compiler.groupby_agg`](https://github.com/modin-project/modin/blob/22e245bfe0c0d9b093aca4e2800baee6ce66f249/modin/backends/base/query_compiler.py#L1112) and passes user-defined function to the compilers. So it seems that there is more general problem. If UDF for `aggregation` is not allowed, it should be explicitly prohibited.

Reply:
UDFs were removed from the kurt implementation. Now function name is passed as apply function argument for Series and for DataFrame modin_interface is used.